paddle/operators/activation_op.h

```diff
@@ -280,6 +280,36 @@ struct BReluGradFunctor : public BaseActivationFunctor<T> {
   }
 };
 
+// relu6(x) = min(max(0, x), 6)
+template <typename T>
+struct Relu6Functor : public BaseActivationFunctor<T> {
+  float threshold;
+
+  // NOTE: Explicit hides the `BaseActivationFunctor<T>::GetAttrs`
+  // not polymorphism for speed.
```

Comment:
I don't think that hiding the `BaseActivationFunctor<T>::GetAttrs` has an effect on speed.  Because it will be expanded in compile time, so it's zero-cost.